parser-preset.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  parserOpts: {
+    headerPattern: /^(\w*)-(\w*) - (.*)$/,
```

Comment:
I think we can use this pattern to store only numbers in second group
```suggestion
    headerPattern: /^(\w*)-(\d*) - (.*)$/,
```